docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -96,3 +83,18 @@ Contributors
 * Kevin Anderson (:ghuser:`kandersolar`)
 * Scott Nelson (:ghuser:`scttnlsn`)
 * Ioannis Sifnaios (:ghuser:`IoannisSifnaios`)
+* Adam R. Jensen (:ghuser:`AdamRJensen`)
+* Anton Driesse (:ghuser:`adriesse`)
+* Will Holmgren (:ghuser:`wholmgren`)
+* Umay Akkoseoglu (:ghuser:`uakkoseo`)
+* Mark Campanelli (:ghuser:`markcampanelli`)
+* :ghuser:`iblasi`
+* Hiromasa Ihara (:ghuser:`miettal`)
+* Yunho Kee (:ghuser:`yhkee0404`)
+* Hamilton Kibbe (:ghuser:`hamiltonkibbe`)
+* Felix Korbelius (:ghuser:`FelixKoTU`)
+* Michael Deceglie (:ghuser:`mdeceglie`)
+* Will Hobbs (:ghuser:`williamhobbs`)
+* Todd Karin (:ghuser:`toddkarin`)
+* Lucas Schneeberger (:ghuser:`lucasschn`)
+* Roma Koulikov (:ghuser:`matsuobasho`)
```

Comment:
I can't comment on the line in question to request a change commit but this ghuser (matsuobasho) is tagged twice. See the first entry below Mark Mikofski